controllers/datafoundation.go

```diff
@@ -79,13 +81,20 @@ func dfSetupWatches(controllerBuilder *builder.Builder) {
 			},
 		),
 	)
+
 	controllerBuilder.
 		Owns(&corev1.Secret{}).
-		Owns(&ocsv1.StorageCluster{}).
 		Owns(&netv1.NetworkPolicy{}).
 		Owns(&corev1.ConfigMap{}).
-		Owns(&opv1a1.ClusterServiceVersion{}, csvPredicates).
-		Owns(&ocsv1.OCSInitialization{})
+		Owns(&opv1a1.ClusterServiceVersion{}, csvPredicates)
+
+	// we only check for storagecluster CRD because other crds are installed as a dependency via ocs-operator
+	if utils.MapCRDAvailability(storageClusterCRD)[storageClusterCRD] {
+		doesCRDExistsAtStart = true
```

Comment:
Is that a global variable? If so it is not differentiated between reconcile calls. We should not do that.